www/modules/custom/borg_project_metrics/borg_project_metrics.module

```diff
@@ -1,13 +1,19 @@
 <?php
+
```

Comment:
Oh; the Drupal phpcs code standards require them;. That is why I do it but of course I will remove;